Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -59,6 +59,7 @@ class BrowserViewController: UIViewController {
     
     // Single data source used for all favorites vcs
     let backgroundDataSource = NTPDataSource()
+    static let feedDataSource = FeedDataSource()
```

Comment:
I did this because I think if we ever supported multiple windows and therefore needed multiple instances of BVC that we'd want only 1 set of data for the feed.

Now that I think about it, a lot of the things in BVC would need to be put somewhere higher, so I will move this back to non-static for the time being